interface-definitions/include/interface/tunnel-remotes.xml.i

```diff
@@ -0,0 +1,19 @@
+<!-- include start from rip/tunnel-remote.xml.i -->
```

Comment:
```
 <!-- include start from rip/tunnel-remote.xml.i -->
```
Replace to the correct include `from`

Reply:
Sorry. I missed that. I hope it's okay now.

Reply:
It still has an incorrect include path.

Reply:
Oh. That was already wrong on the "tunnel-remote.xml.i" I copied from.
I immediately correct that.

Am I allowed to correct both in one patch or do I have to create a different PR for correcting the other file?
I suppose the 2nd.